From 60b28e844429cfc70a538b582deb78c9f7c63ca4 Mon Sep 17 00:00:00 2001
From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Date: Wed, 25 Sep 2019 14:09:28 +0300
Subject: [PATCH] Add vspfilter config for other salvator-xs-h3* machines

These are just the copy from the original
gstvspfilter-salvator-x_r8a7795.conf

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 .../gstvspfilter-salvator-xs-4x2g_r8a7795.conf                          | 2 ++
 .../gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs_r8a7795.conf | 2 ++
 3 files changed, 6 insertions(+)
 create mode 100644 meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs-4x2g_r8a7795.conf
 create mode 100644 meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs_r8a7795.conf

diff --git a/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs-4x2g_r8a7795.conf b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs-4x2g_r8a7795.conf
new file mode 100644
index 0000000..586b13b
--- /dev/null
+++ b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs-4x2g_r8a7795.conf
@@ -0,0 +1,2 @@
+input-device-name=/dev/video20
+output-device-name=/dev/video21
diff --git a/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs_r8a7795.conf b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs_r8a7795.conf
new file mode 100644
index 0000000..586b13b
--- /dev/null
+++ b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-xs_r8a7795.conf
@@ -0,0 +1,2 @@
+input-device-name=/dev/video20
+output-device-name=/dev/video21
-- 
2.7.4

